README.md

```diff
@@ -1,135 +1,54 @@
-Chitter Challenge
-=================
+# Chitter
 
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+![chitter](docs/chitter.png)
 
-Challenge:
--------
+---------
+### Functions
 
-As usual please start by forking this repo.
+1. Sign up. Prevent register the same email or username twice, with error message in red.
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+2. Sign in. Reject wrong email or password, with error message in red.
 
-Features:
--------
+3. View messages without the need to sign in. Messages are listed from latest to oldest.
 
-```
-STRAIGHT UP
+4. Can post message after sign in. 
 
-As a Maker
-So that I can let people know what I am doing  
-I want to post a message (peep) to chitter
+5. Can send email to tagged user
 
-As a maker
-So that I can see what others are saying  
-I want to see all peeps in reverse chronological order
+---------
+### How to use / test
 
-As a Maker
-So that I can better appreciate the context of a peep
-I want to see the time at which it was made
+1. [Setup](docs/setup.md)
+2. Run command `rackup`
+3. In web browser, enter i.e. "localhost:9292" (or the port provided from previous step)
+4. Sign up a few users (some with your real email addresses, best with gmail), play around to register same email and username twice to see the error messages
+5. Sign out
+6. Sign in with registered users, play around with wrong email or password
+7. Chit a few messages, latest will appear at the top
+8. Chit messages with @username (replace username with real usernames registered), choose the ones with real email addresses. You should receive email notifications (check junk if not)
 
-As a Maker
-So that I can post messages on Chitter as me
-I want to sign up for Chitter
+*Tried with hotmail, seems not getting the notification. With gmail, getting it straightaway in junk folder.*
```

Comment:
I've tried signing-up using Gmail, but for some reason did not get an e-mail - not in junk mail folder either.